Curve-finance-suite/Create-lockevent/README.md

```diff
@@ -0,0 +1,26 @@
+# Create lock Agent
+
+## Description
+
+This agent detects lock creation in Voting Escrow contract.
+
+## Supported Chains
+
+- Ethereum
+
+## Alerts
+
+- CURVE-7
+  - Fired when a Deposit event is emitted with type set to 1.
+  - Severity is always set to "low".
+  - Type is always set to "suspicious".
+  - Metadata includes: 
+    - `from`: address of the account calling create lock function.
+    - `value`: The amount of CRV locked.
+    - `locktime`: lock duration. 
```

Comment:
Can we check if this is the `lock duration` or `when the lock ends`

Reply:
Ah yes it's the time when to unlock the CRV tokens. Readme updated.